src/infrastructure/middleware/authorization/whiteList/entriesUtils.ts

```diff
@@ -0,0 +1,7 @@
+import { StringUtil } from "../../../../domain/shared/utils/StringUtil";
+
+export class EntriesUtils {
```

Comment:
We are using Util or Utils? I mean, SomethingUtil or SomethingUtils. Independent you choose, please fix the other one classes.